docs/Documentation/Environment/shell.md

```diff
@@ -178,9 +185,9 @@ alias mlist='module list 2>&1 |  egrep -v "Current|No modules loaded" | sed "s/.
 
 
 #### Loading modules
-Most HPC platforms run module systems.  When you load a module changes some environmental variable setting.  Often PATH and LD_LIBARAY_PATH are changed.  In general loading a module will allow you to use a particular application or library.  
+Most HPC platforms run module systems.  When you load a module, it will change some environmental variable setting.  Often, PATH and LD_LIBARAY_PATH are changed.  In general, loading a module will allow you to use a particular application or library.  
```

Comment:
```suggestion
Most HPC platforms run module systems.  When you load a module, it will change some environmental variable settings.  Often, PATH and LD_LIBARAY_PATH are changed.  In general, loading a module will allow you to use a particular application or library.  
```